### PR TITLE
Writing coverage for marketing model, increasing coverage to 33%

### PR DESCRIPTION
**Pull request by leonprou**
##### Description
Adding tests to increase code coverage from 28% to 33%. Testing marketing module.

##### Checklist

- [x] linter status: 100% pass.
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)
marketing

##### Testing
This PR is increasing test coverage. Only one little refactor was made.

#####  Refers/Fixes
[#408](https://github.com/gitcoinco/web/issues/408)

### CODE REVIEW

**Comment by leonprou** [comment]
@owocki hey, I'd like to merge it but it seems like the tests are failing again with same error as before
 
> _pytest.vendored_packages.pluggy.PluginValidationError: Plugin 'ethereum' could not be loaded: (eth-utils 1.0.0b2 (/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages), Requirement.parse('eth-utils<2.0.0,>=1.0.1'), {'hexbytes', 'eth-abi'})!

**Comment by leonprou** [comment]
I think `expitation_start_work` got broken, it's uses `created_at` [here](https://github.com/gitcoinco/web/blob/master/app/marketing/management/commands/expiration_start_work.py#L70). While `Interest` class has been [changed](https://github.com/gitcoinco/web/blob/master/app/dashboard/models.py#L591) to have `created` property.  Found it running the tests after merge.

**Comment by owocki** [comment]
> I think expitation_start_work got broken, it's uses created_at here. While Interest class has been changed to have created property. Found it running the tests after merge.

shoot.. you are right.. 9c82c4ff29de10f9294dde2b2237bb073e8ea408 should fix that

**Comment by owocki** [comment]
> @owocki hey, I'd like to merge it but it seems like the tests are failing again with same error as before

IIRC, you and @mbeacom chatted about this on slack already, right?

**Comment by owocki** [comment]
just poked through the new tests.. 💯 thanks so much!

**Comment by leonprou** [comment]
> IIRC, you and @mbeacom chatted about this on slack already, right?

Yeah, we talked. it's good now 👍 

> just poked through the new tests.. 💯 thanks so much!

You're welcome 😄, learned a thing or two writing the tests. I think more bounties like this are needed,  I communicate some more thoughts later.

For now `expiration_start_work` implementation changed so the tests not pass, I'll look at that at the evening I guess.

**Comment by owocki** [comment]
> For now expiration_start_work implementation changed so the tests not pass, I'll look at that at the evening I guess.

did you mean that my commit (hash pasted above in my previous comment) broke the tests?

**Comment by leonprou** [comment]
Oh no, some logic was changed before.. I assumed it was intentionally

**Comment by owocki** [comment]
> Oh no, some logic was changed before.. I assumed it was intentionally

The logic that was changed in https://github.com/gitcoinco/web/commit/9c82c4ff29de10f9294dde2b2237bb073e8ea408 is just that if a comment happens before the 'start work' button is clicked it is disregarded.. 

Let me know if you have other questions to get the tests working agin

**Comment by leonprou** [comment]
> The logic that was changed in 9c82c4f is just that if a comment happens before the 'start work' button is clicked it is disregarded..

Yes I solved that. Ok I think that this can be merged 😄, the last thing is about the linter? Which one you are using?

**Comment by mbeacom** [comment]
@leonprou I regularly use `flake8`, `pylint`, and `pydocstyle` along with a combination of plugins.

**Comment by owocki** [comment]
i use 

```isort --recursive --atomic```

but i'm not as good of a citizen as @mbeacom 

@mbeacom maybe we should update the readme with some agreed-upon linting commands so that maintainers can be expected to lint their own PRs?

**Comment by mbeacom** [comment]
@owocki I'm all for updating the docs to reflect a standard linting practice.  I've been meaning to ask you about your thoughts on:  https://stickler-ci.com/ - It's free for OSS.  We could simply use their linter without using the `fixer` functionality to identify spots throughout a PR that don't adhere to the linting configuration for both python (flake8) and js (eslint).

**Comment by owocki** [comment]
certainly open to trying it.. hopefully between travis + codecov + stickler its not too overwhelming for new contributors

**Comment by mbeacom** [comment]
@leonprou Is this ready to review and test?

**Comment by leonprou** [comment]
Yes it's ready
There's a problem in master I think, a linting  (isort) is failing.

**Comment by mbeacom** [comment]
@leonprou Yeah, that's fine... It's an isort failure on an import in `sync_slack.py` - All good.

Reviewing this now.

File: app/marketing/management/commands/assemble_leaderboards.py [hunk 1]
```diff
@@ -33,28 +33,31 @@
...
-    'yearly_all': {},
-    'yearly_payers': {},
-    'yearly_earners': {},
-    'all_fulfilled': {},
-    'all_all': {},
-    'all_payers': {},
-    'all_earners': {},
-}
+def default_ranks():
+    return {
```
Comment (mbeacom) [inline app/marketing/management/commands/assemble_leaderboards.py]: 👍

File: app/marketing/tests/management/commands/test_assemble_leaderboards.py [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+            expires_date=datetime(2008, 11, 30),
+            idx_project_length=5,
+            project_length='Months',
+            bounty_type='Feature',
+            experience_level='Intermediate',
+            raw_data={},
+            idx_status='submitted',
+            current_bounty=True
+        )
+        bounty_fulfillment = BountyFulfillment.objects.create(
```
Comment (mbeacom) [inline app/marketing/tests/management/commands/test_assemble_leaderboards.py]: Can you remove the unused assignment of `bounty_fulfillment` here?

File: app/marketing/tests/management/commands/test_assemble_leaderboards.py [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+        sum_bounties(self.bounty, ['fred'])
+
+        print(assemble_leaderboards.ranks)
+
+        assert assemble_leaderboards.ranks['all_all']['fred'] == 3
+        assert assemble_leaderboards.ranks['weekly_all']['fred'] == 3
+        assert assemble_leaderboards.ranks['monthly_all']['fred'] == 3
+        assert assemble_leaderboards.ranks['yearly_all']['fred'] == 3
+
+        assert len(assemble_leaderboards.ranks['weekly_fulfilled']) == 0
```
Comment (mbeacom) [inline app/marketing/tests/management/commands/test_assemble_leaderboards.py]: Opposed to `len()`, can you use either `assert any(dict) is False` or `assert not dict`

File: app/marketing/tests/management/commands/test_assemble_leaderboards.py [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+            bounty=bounty,
+            profile=fulfiller_profile,
+        )
+
+        self.bounty = bounty
+        self.tip = tip
+
+    def test_sum_tips(self):
+        """Test sum tips of assemble leaderboards."""
+
```
Comment (mbeacom) [inline app/marketing/tests/management/commands/test_assemble_leaderboards.py]: You can remove this extra line.

File: app/marketing/tests/management/commands/test_assemble_leaderboards.py [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+        assert assemble_leaderboards.ranks['monthly_earners']['john'] == 7
+        assert assemble_leaderboards.ranks['monthly_all']['john'] == 7
+
+        assert assemble_leaderboards.ranks['yearly_fulfilled']['john'] == 7
+        assert assemble_leaderboards.ranks['yearly_earners']['john'] == 7
+        assert assemble_leaderboards.ranks['yearly_all']['john'] == 7
+
+    def test_sum_bounties(self):
+        """Test sum tips of bounties leaderboards."""
+
```
Comment (mbeacom) [inline app/marketing/tests/management/commands/test_assemble_leaderboards.py]: You can remove this extra line.

File: app/marketing/tests/management/commands/test_sync_keywords.py [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+            experience_level='Intermediate',
+            raw_data={},
+            idx_status='submitted',
+            bounty_owner_email='jack@bar.com',
+            current_bounty=True
+        )
+
+    def test_handle_simple(self):
+        """Test command sync keywords."""
+
```
Comment (mbeacom) [inline app/marketing/tests/management/commands/test_sync_keywords.py]: You can remove this extra line.

File: app/marketing/tests/management/commands/test_sync_keywords.py [hunk 7]
```diff
@@ -0,0 +1,133 @@
...
+        Command().handle()
+
+        assert Keyword.objects.all().count() == 4
+
+
```
Comment (mbeacom) [inline app/marketing/tests/management/commands/test_sync_keywords.py]: You can remove this extra line.

File: app/marketing/tests/test_marketing_utils.py [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+        )
+        EmailSubscriber.objects.create(
+            email='emailSubscriber3@gitcoin.co',
+            source='mysource',
+            preferences={
+                'level': 'nothing'
+            }
+        )
+
+
```
Comment (mbeacom) [inline app/marketing/tests/test_marketing_utils.py]: You can remove this extra line.

File: app/marketing/tests/test_marketing_utils.py [hunk 9]
```diff
@@ -0,0 +1,92 @@
...
+            priv='priv2'
+        )
+
+        assert get_or_save_email_subscriber('emailSubscriber1@gitcoin.co', 'secondsource') == 'priv1'
+
+    def test_save_get_or_save_email_subscriber_get(self):
+        """Test the marketing util get_or_save_email_subscriber method."""
+        self.assertIsNotNone(get_or_save_email_subscriber('newemail@gitcoin.co',
+            'mysource', send_slack_invite=False))
+        assert EmailSubscriber.objects.filter().count() == 4
```
Comment (mbeacom) [inline app/marketing/tests/test_marketing_utils.py]: Can you add a new line here?

File: app/marketing/tests/management/commands/test_expiration.py [hunk 10]
```diff
@@ -0,0 +1,109 @@
...
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""
+from datetime import date, datetime, timedelta
```
Comment (mbeacom) [inline app/marketing/tests/management/commands/test_expiration.py]: Can you remove the unused `date` import here?

File: app/marketing/tests/management/commands/test_expiration.py [hunk 11]
```diff
@@ -0,0 +1,109 @@
...
+from unittest.mock import patch
+
+from django.test import TestCase
+from django.utils import timezone
+
+from dashboard.models import Bounty, BountyFulfillment, Profile, Tip
```
Comment (mbeacom) [inline app/marketing/tests/management/commands/test_expiration.py]: Can you remove the unused `Tip` import here?

File: app/marketing/tests/management/commands/test_expiration.py [hunk 12]
```diff
@@ -0,0 +1,109 @@
...
+            idx_status='submitted',
+            bounty_owner_email='john@bar.com',
+            current_bounty=True
+        )
+        fulfiller_profile = Profile.objects.create(
+            data={},
+            handle='fred',
+            email='fred@bar.com'
+        )
+        bounty_fulfillment = BountyFulfillment.objects.create(
```
Comment (mbeacom) [inline app/marketing/tests/management/commands/test_expiration.py]: Can you remove the unused assignment here?

File: app/marketing/tests/management/commands/test_expiration.py [hunk 13]
```diff
@@ -0,0 +1,109 @@
...
+            expires_date=timezone.now() + timedelta(days=1, hours=1),
+            idx_project_length=5,
+            project_length='Months',
+            bounty_type='Feature',
+            experience_level='Intermediate',
+            raw_data={},
+            idx_status='submitted',
+            current_bounty=True
+        )
+        bounty_fulfillment = BountyFulfillment.objects.create(
```
Comment (mbeacom) [inline app/marketing/tests/management/commands/test_expiration.py]: Can you remove the unused assignment here?

File: app/marketing/tests/management/commands/test_expiration_start_work.py [hunk 14]
```diff
@@ -0,0 +1,156 @@
...
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""
+from datetime import date, datetime, timedelta
```
Comment (mbeacom) [inline app/marketing/tests/management/commands/test_expiration_start_work.py]: Can you removed the unused `date` import here?

File: app/marketing/tests/management/commands/test_expiration_start_work.py [hunk 15]
```diff
@@ -0,0 +1,156 @@
...
+
+comments_warning = [
+    {
+        'user': {
+            'login': 'fred'
+        },
+        'created_at': (timezone.now() - timedelta(days=5)).strftime('%Y-%m-%dT%H:%M:%SZ')
+    }
+]
+
```
Comment (mbeacom) [inline app/marketing/tests/management/commands/test_expiration_start_work.py]: Can you add a new line here?

File: app/marketing/tests/management/commands/test_expiration_start_work.py [hunk 16]
```diff
@@ -0,0 +1,156 @@
...
+        """Test command expiration start work with not interests."""
+        Command().handle()
+
+        assert mock_bounty_startwork_expired.call_count == 0
+        assert mock_bounty_startwork_expire_warning.call_count == 0
+
+    @patch('marketing.management.commands.expiration_start_work.get_issue_comments', return_value=comments_expired)
+    @patch('marketing.management.commands.expiration_start_work.bounty_startwork_expire_warning')
+    @patch('marketing.management.commands.expiration_start_work.bounty_startwork_expired')
+    def test_handle_expired(self, mock_bounty_startwork_expired, mock_bounty_startwork_expire_warning, mock_get_issue_comments):
```
Comment (mbeacom) [inline app/marketing/tests/management/commands/test_expiration_start_work.py]: Can you remove the unused argument `mock_get_issue_comments`?

File: app/marketing/tests/management/commands/test_expiration_start_work.py [hunk 17]
```diff
@@ -0,0 +1,156 @@
...
+        Command().handle()
+
+        assert mock_bounty_startwork_expire_warning.call_count == 0
+        assert mock_bounty_startwork_expired.call_count == 1
+
+
+    @patch('marketing.management.commands.expiration_start_work.get_issue_comments', return_value=comments_warning)
+    @patch('marketing.management.commands.expiration_start_work.bounty_startwork_expire_warning')
+    @patch('marketing.management.commands.expiration_start_work.bounty_startwork_expired')
+    def test_handle_expire_warning(self, mock_bounty_startwork_expired, mock_bounty_startwork_expire_warning, mock_get_issue_comments):
```
Comment (mbeacom) [inline app/marketing/tests/management/commands/test_expiration_start_work.py]: Can you remove the unused argument `mock_get_issue_comments`?

File: app/marketing/tests/management/commands/test_expiration_start_work.py [hunk 18]
```diff
@@ -0,0 +1,156 @@
...
+
+        bounty.interested.add(interest)
+        bounty.save()
+
+        Command().handle()
+
+        assert mock_bounty_startwork_expire_warning.call_count == 0
+        assert mock_bounty_startwork_expired.call_count == 1
+
+
```
Comment (mbeacom) [inline app/marketing/tests/management/commands/test_expiration_start_work.py]: You can remove this extra line.

File: app/marketing/tests/management/commands/test_expiration_tip.py [hunk 19]
```diff
@@ -0,0 +1,49 @@
...
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""
+from datetime import date, datetime, timedelta
```
Comment (mbeacom) [inline app/marketing/tests/management/commands/test_expiration_tip.py]: Can you remove the unused `date` and `datetime` imports here?

File: app/marketing/tests/management/commands/test_remarket_tweet.py [hunk 20]
```diff
@@ -0,0 +1,69 @@
...
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""
+from datetime import date, datetime, timedelta
```
Comment (mbeacom) [inline app/marketing/tests/management/commands/test_remarket_tweet.py]: Can you remove this import line?

File: app/marketing/tests/management/commands/test_remarket_tweet.py [hunk 21]
```diff
@@ -0,0 +1,69 @@
...
+from unittest.mock import patch
+
+from django.test import TestCase
+from django.utils import timezone
+
+from dashboard.models import Bounty
```
Comment (mbeacom) [inline app/marketing/tests/management/commands/test_remarket_tweet.py]: Can you remove this import?

File: app/marketing/tests/management/commands/test_remarket_tweet.py [hunk 22]
```diff
@@ -0,0 +1,69 @@
...
+
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""
+from datetime import date, datetime, timedelta
+from unittest.mock import patch
+
+from django.test import TestCase
+from django.utils import timezone
```
Comment (mbeacom) [inline app/marketing/tests/management/commands/test_remarket_tweet.py]: Can you remove this import?

File: app/marketing/tests/management/commands/test_roundup.py [hunk 23]
```diff
@@ -0,0 +1,71 @@
...
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""
+from datetime import date, datetime, timedelta
```
Comment (mbeacom) [inline app/marketing/tests/management/commands/test_roundup.py]: Can you remove this import?

File: app/marketing/tests/management/commands/test_roundup.py [hunk 24]
```diff
@@ -0,0 +1,71 @@
...
+from unittest.mock import patch
+
+from django.test import TestCase
+from django.utils import timezone
```
Comment (mbeacom) [inline app/marketing/tests/management/commands/test_roundup.py]: Can you remove this import?

File: app/marketing/tests/management/commands/test_roundup.py [hunk 25]
```diff
@@ -0,0 +1,71 @@
...
+            newsletter=True
+        )
+        EmailSubscriber.objects.create(
+            email='paul@bar.com',
+            source='mysource'
+        )
+
+    @patch('time.sleep')
+    @patch('marketing.management.commands.roundup.weekly_roundup')
+    def test_handle_no_options(self, mock_weekly_roundup, mock_time):
```
Comment (mbeacom) [inline app/marketing/tests/management/commands/test_roundup.py]: Can you remove the unused argument: `mock_time` here?

File: app/marketing/tests/management/commands/test_roundup.py [hunk 26]
```diff
@@ -0,0 +1,71 @@
...
+        """Test command roundup when live option is False."""
+        Command().handle(exclude_startswith=None, filter_startswith=None, live=False)
+
+        assert mock_weekly_roundup.call_count == 0
+
+    @patch('time.sleep')
+    @patch('marketing.management.commands.roundup.weekly_roundup')
+    def test_handle_with_options(self, mock_weekly_roundup, mock_time):
```
Comment (mbeacom) [inline app/marketing/tests/management/commands/test_roundup.py]: Can you remove the unused argument: `mock_time` here?

File: app/marketing/tests/management/commands/test_sync_keywords.py [hunk 27]
```diff
@@ -0,0 +1,133 @@
...
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""
+from datetime import date, datetime, timedelta
```
Comment (mbeacom) [inline app/marketing/tests/management/commands/test_sync_keywords.py]: Can you remove the `date` import here?

File: app/marketing/tests/management/commands/test_sync_keywords.py [hunk 28]
```diff
@@ -0,0 +1,133 @@
...
+from unittest.mock import patch
```
Comment (mbeacom) [inline app/marketing/tests/management/commands/test_sync_keywords.py]: Can you remove this import?

File: app/marketing/tests/management/commands/test_sync_keywords.py [hunk 29]
```diff
@@ -0,0 +1,133 @@
...
+
+from django.test import TestCase
+from django.utils import timezone
+
+from dashboard.models import Bounty, BountyFulfillment, Profile, Tip
```
Comment (mbeacom) [inline app/marketing/tests/management/commands/test_sync_keywords.py]: Can you remove the unused `Profile` and `Tip` imports here?

File: app/marketing/tests/management/commands/test_sync_keywords.py [hunk 30]
```diff
@@ -0,0 +1,133 @@
...
+            raw_data={},
+            idx_status='submitted',
+            bounty_owner_email='john@bar.com',
+            current_bounty=True,
+            metadata={
+                'issueKeywords': 'ethereum, blockchain,'
+            }
+        )
+
+        bounty_fulfillment = BountyFulfillment.objects.create(
```
Comment (mbeacom) [inline app/marketing/tests/management/commands/test_sync_keywords.py]: Can you remove the unused assignment here?

File: app/marketing/tests/management/commands/test_sync_keywords.py [hunk 31]
```diff
@@ -0,0 +1,133 @@
...
+            fulfiller_address='0x0000000000000000000000000000000000000000',
+            fulfiller_email='fred@bar.com',
+            fulfiller_github_username='fred',
+            fulfiller_name='Fred',
+            bounty=bounty
+        )
+
+        bounty_fulfillment = BountyFulfillment.objects.create(
```
Comment (mbeacom) [inline app/marketing/tests/management/commands/test_sync_keywords.py]: Can you remove the unused assignment here?

File: app/marketing/tests/management/commands/test_sync_keywords.py [hunk 32]
```diff
@@ -0,0 +1,133 @@
...
+
+from dashboard.models import Bounty, BountyFulfillment, Profile, Tip
+from marketing.management.commands.sync_keywords import Command
+from marketing.models import Keyword
+
+
+class TestSyncKeywords(TestCase):
+    """Define tests for sync keywords."""
+
+    def setUp(test):
```
Comment (mbeacom) [inline app/marketing/tests/management/commands/test_sync_keywords.py]: Can you switch this to `def setUp(self):` ?

File: app/marketing/tests/test_marketing_utils.py [hunk 33]
```diff
@@ -0,0 +1,92 @@
...
+        )
+        Stat.objects.create(
+            key='mykey',
+            val=2
+        )
+
+    def test_get_stat(self):
+        """Test the marketing util get_stat method."""
+        assert get_stat('mykey') == 2
+
```
Comment (mbeacom) [inline app/marketing/tests/test_marketing_utils.py]: Can you add another new line between these?

File: app/marketing/tests/test_marketing_utils.py [hunk 34]
```diff
@@ -0,0 +1,92 @@
...
+            source='mysource',
+            preferences={
+                'level': 'nothing'
+            }
+        )
+
+
+    def test_should_suppress_notification_email(self):
+        """Test the marketing util test_should_suppress_notification_email method."""
+        assert should_suppress_notification_email('emailSubscriber1@gitcoin.co') == False
```
Comment (mbeacom) [inline app/marketing/tests/test_marketing_utils.py]: Can you change these boolean assertions to `assert func() is False` and `assert func() is True` **or** `assert func()` and `assert not func()`

File: app/marketing/tests/test_marketing_utils.py [hunk 35]
```diff
@@ -0,0 +1,92 @@
...
+            source='secondsource',
+            priv='priv2'
+        )
+
+        assert get_or_save_email_subscriber('emailSubscriber1@gitcoin.co', 'secondsource') == 'priv1'
+
+    def test_save_get_or_save_email_subscriber_get(self):
+        """Test the marketing util get_or_save_email_subscriber method."""
+        self.assertIsNotNone(get_or_save_email_subscriber('newemail@gitcoin.co',
+            'mysource', send_slack_invite=False))
```
Comment (mbeacom) [inline app/marketing/tests/test_marketing_utils.py]: Can you carry this indent over to hang below `(get_or_save`?

**Comment by mbeacom** [comment]
Overall, lgtm - Just a few miscellaneous syntax changes.

File: app/marketing/tests/management/commands/test_sync_keywords.py (hunk 32 above)
Comment (leonprou) [reply app/marketing/tests/management/commands/test_sync_keywords.py]: sure 👍

File: app/marketing/tests/management/commands/test_send_subscription_mails.py [hunk 36]
```diff
@@ -0,0 +1,56 @@
...
+from dashboard.models import Bounty, Subscription
+from marketing.management.commands.send_subscription_mails import Command
+from marketing.models import Keyword
+
+
+class TestSendSubsciptionMails(TestCase):
+    """Define tests for roundup send subscription mails."""
+
+    def test_handle(self):
+        """Test command send subscription mails."""
```
[the remark this answers is not in the document]
Comment (leonprou) [reply app/marketing/tests/management/commands/test_send_subscription_mails.py]: thanks, forgot to test that one 👍

File: app/marketing/tests/test_marketing_utils.py (hunk 33 above)
Comment (leonprou) [reply app/marketing/tests/test_marketing_utils.py]: you want to separate the classes by 2 lines, right?

**Comment by owocki** [comment]
lets get this shipped this week!

File: app/marketing/tests/test_marketing_utils.py (hunk 33 above)
Comment (mbeacom) [reply app/marketing/tests/test_marketing_utils.py]: Yes @leonprou - Two new lines between module level classes and methods.

```python
import thing


class Example:
    """Some docstring."""

    def test(self):
        """Some test."""
        return 'test'

    def test_again(self):
        """Some test again."""
        return 'test again'


class ExampleAgain:
    """Some docstring again."""

    def test(self):
        """Some test."""
        return 'test'


def some_test():
    """Some test."""
    return 'some test'

```

**Pull request merged**